### ISSUE TITLE
Licence documentation

### ISSUE THREAD

**Issue by xray7224** [xray7224/PyPump#136]
There doesn't seem to be any licences for the documentation, we should fix this.

Which licence should we choose?

**Comment by moggers87** [comment]
Personally, I think the docs should be licensed the same as the codebase - GPLv3 or later in this case.

**Comment by dper** [comment]
It is simple if the docs and code have the same license.

**Comment by moggers87** [comment]
@xray7224 @kabniel any objections to my suggestion?

**Comment by xray7224** [comment]
Nope, none. All that would be required in that case to close this is to add the licence to the header like the python files.

**Comment by kabniel** [comment]
Sounds good to me

**Comment by cwebber** [comment]
I'm good with GPLv3+
